config/tips.txt

```diff
@@ -163,9 +163,12 @@ As a Gang Boss, don't wait too long to promote lieutenants! If you get caught by
 As an Abductor, you can select where your victims will be sent on the ship control console.
 As an Abductor Agent, the combat mode vest has much higher resistance to every kind of weapon, and your helmet prevents the AI from tracking you.
 As an Abductor, the baton is set up to allow you to rapidly stun, sleep, and cuff: From stun mode, it cyles to sleep, then to cuff, allowing you to render a target helpless and unable to communicate very rapidly.
-As a Revenant, the Chaplain is your worst enemy, as he can damage you with his null rod and make large swaths of the station impassable with holy water.
+As a Revenant, the Chaplain is your worst enemy, as they can damage you with his null rod and make large swaths of the station impassable with holy water.
 As a Revenant, your essence is also your health, so revealing yourself in front of humans to harvest the essence of the living is much safer if you've already stocked up on essences from poorly guarded corpses.
-As a Revenant, the illness inflicted by Blight can be cured with sleep but a sleeping human is always vulnerable to a revenant's harvest, even if they're completely healthy.
```

Comment:
please don't sleep in front of a revenant you will die and you don't need to do that